### DISCUSSION TITLE
Can I close buffers from the buffer list?

### DISCUSSION THREAD

**Discussion by despairblue** [nvim-telescope/telescope.nvim#1512] [Q&A] [answered]
This would be a nice UX to clean open buffers when using `set hidden`.

![image](https://user-images.githubusercontent.com/927609/143867193-10bdbb6e-fa01-4fda-802e-b486c46ad666.png)

**Comment by Conni2461** [comment, marked as answer]
We have a action for that that isnt mapped. This will map it to `<c-d>`

```lua
local actions = require("telescope.actions")

require('telescope').setup{
  pickers = {
    buffers = {
      mappings = {
        i = {
          ["<c-d>"] = actions.delete_buffer
        },
      }
    },
  },
}
```

**Reply by despairblue** [reply]
This is amazing. Thank you!
